contents/product-engineers/llm-product-metrics.md

```diff
@@ -0,0 +1,157 @@
+---
+date: 2024-05-03
+title: 'Product metrics to track for LLM apps'
+author:
+  - lior-neu-ner
+featuredImage: >-
+  https://res.cloudinary.com/dmukukwp6/image/upload/posthog.com/contents/images/blog/posthog-marketing/marketing-hog.jpg
+featuredImageType: full
+category: Engineering
+tags:
+  - Guides
+  - Product metrics
+crosspost:
+  - Blog
+---
+
+import { ProductScreenshot } from 'components/ProductScreenshot'
+
+LLM-powered apps differ in three key ways when compared to regular SaaS apps:
+
+1. Their costs increase as as app usage increases.
+2. Request latency spikes and errors are more common.
+3. You're not always sure if your LLM output is what was expected.
+
+This means that in addition to [regular product metrics](https://posthog.com/product-engineers/product-health-metrics), you need to monitor LLM-specific ones too. To understand which ones to track, we've put together a list of useful metrics to track.
+
+The metrics are grouped into three categories: [cost](#cost-related-metrics), [usage](#usage-metrics), and [debugging](#debug-metrics).
+
```

Comment:
Do you want to add a callout here to say "to easily capture the data for these metrics, use our Langfuse and Helicone integrations"?

Reply:
Also, does the data captured by these cover all the metrics?

Reply:
> Also, does the data captured by these cover all the metrics?

yes!